.github/workflows/windows-agent.yml

```diff
@@ -61,27 +68,10 @@ jobs:
           cd build_windows
           tests/ut_agent
 
-      - name: Zip agent
-        run: |
-          $files_to_compress = "agent\conf\centagent.reg", "build_windows\agent\Release\centagent.exe"
-          Compress-Archive -Path $files_to_compress -DestinationPath centreon-monitoring-agent.zip
-
-      - name: Save agent package in cache
-        uses: actions/cache/save@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
-        with:
-          path: centreon-monitoring-agent.zip
-          key: ${{ github.run_id }}-${{ github.sha }}-CMA-${{ github.head_ref || github.ref_name }}
-          enableCrossOsArchive: ${{ true }}
-
       - name: Upload package artifacts
-        if: |
-          github.event_name != 'workflow_dispatch' &&
-          contains(fromJson('["stable"]'), needs.get-version.outputs.stability) &&
-          ! cancelled() &&
-          ! contains(needs.*.result, 'failure') &&
-          ! contains(needs.*.result, 'cancelled')
+        if: inputs.installer_in_artifact == true
```

Comment:
stable releases will not upload installer in artifact automatically, is it what you want ?